packages/frontend/src/pages/common/Page.tsx

```diff
@@ -1,20 +1,37 @@
 import React, { ReactNode } from 'react'
+import { Footer } from './Footer'
 import { Head, HeadProps } from './Head'
+import { Navbar } from './Navbar'
 
 interface Props extends HeadProps {
   scripts: string[]
   children: ReactNode
+  navbarSearch?: boolean
 }
 
-export function Page({ children, scripts, ...head }: Props) {
+export function Page({
+  children,
+  scripts,
+  navbarSearch = true,
```

Comment:
I don't like when optional parameters default to something other than undefined, because it introcuded ambiguity. Consider the following:
```tsx
<Page />
<Page navbarSearch />
```
Instead you could have a parameter like `withoutSearch` that defaults to undefined.

Reply:
In general I agree, but in this case we only have one Page that does not need a navbar, which results in less code overall.